npm/test/index.test.ts

```diff
@@ -116,3 +116,35 @@ test("renderModules does not call authorisation when dialog box is not present o
 
   keycloakInstance.mockRestore()
 })
+
+test("renderModules should initialize the tna-multi-select-search element on the page", async () => {
```

Comment:
Maybe reword the test slightly: `renderModules should initialise the multi-select search module if the tna-multi-select-search element is present on the page`